Makefile

```diff
@@ -59,6 +62,10 @@ licenses: ## Verifies dependency licenses
 	go mod download
 	! go-licenses csv ./... | grep -v -e 'MIT' -e 'Apache-2.0' -e 'BSD-3-Clause' -e 'BSD-2-Clause' -e 'ISC' -e 'MPL-2.0'
 
+build: ## Build controller and webhook and publich to local docker registry
```

Comment:
What's the usecase here?

Reply:
I think `make apply` should cover your usecase if you're pointing to a local cluster.

Reply:
`make apply` will try to deploy to the local or not local cluster.
Use case for `make build` was:
* Build locally
* Save images to the `targ.gz`
* Upload archive to the server with proper IAM
* Use images for testing purposes

Unfortunately `make apply` won't allow to do it. It is possible to get image archive via `ko`, having make target seems to be more convenient

Reply:
Also comes handy when "run go build like we would have in ci" needed for testing purposes

Reply:
Another usecase is to prepare images for pushing to the remote repo without executing any helm operations

Reply:
Ko supports storing images in a local repository as part of make apply. I'd prefer to leave this target out, since it won't be tested regularly by the team, and will be difficult to verify if its working.

Reply:
@ellistarn Sure, will remove. What about leaving doc part with the local image build how-to (still for purposes when there is a testing cluster running in the AWS, but karpenter deployed not by `make apply`)?
```
KO_DOCKER_REPO=ko.local ko build -P github.com/aws/karpenter/cmd/controller
KO_DOCKER_REPO=ko.local ko build -P github.com/aws/karpenter/cmd/webhook
```